docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -204,14 +269,23 @@ argument to this attribute or a list of arguments:
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompilationTask
 // ...
 
-val compileKotlin: KotlinCompilationTask<*> by tasks
+kotlin {
+    compilerOptions {
+        // Single experimental argument
+        freeCompilerArgs.add("-Xexport-kdoc")
+
+        // Single additional argument
+        freeCompilerArgs.add("-Xno-param-assertions")
 
-// Single experimental argument
-compileKotlin.compilerOptions.freeCompilerArgs.add("-Xexport-kdoc")
-// Single additional argument, can be a key-value pair
-compileKotlin.compilerOptions.freeCompilerArgs.add("-Xno-param-assertions")
-// List of arguments
-compileKotlin.compilerOptions.freeCompilerArgs.addAll(listOf("-Xno-receiver-assertions", "-Xno-call-assertions"))
+        // List of arguments
+        freeCompilerArgs.addAll(
+            listOf(
+                "Xno-receiver-assertions",
+                "Xno-call-assertions"
```

Comment:
```suggestion
                "-Xno-receiver-assertions",
                "-Xno-call-assertions"
```

Reply:
Aren't we removing these code lines if we change the whole snippet to:

`compilerOptions { apiVersion.set(KotlinVersion.KOTLIN_2_0) jvmTarget.set(JvmTarget.JVM_1_8) }`